worker-configs/src/main/java/com/github/workerframework/configs/RabbitConfiguration.java

```diff
@@ -51,6 +51,12 @@ public class RabbitConfiguration
     @NotNull
     @Size(min = 1)
     private String rabbitProtocol;
+    /**
+     * The TLS protocol version, only respected if 'rabbitProtocol' is 'amqps'
+     */
+    @NotNull
+    @Size(min = 1)
```

Comment:
Making a new configuration setting mandatory like this is a breaking change.  New configuration settings need to be optional.

Reply:
Fortunately the update made to the default configuration file means that most consumers are not impacted.  The dependency update builds of projects that are not using the default configuration file are failing:

- [CAFDataProcessing/worker-languagedetection](https://github.com/[CAFDataProcessing/worker-languagedetection](https://github.com/CAFDataProcessing/worker-languagedetection))
- [JobService/job-service](https://github.com/JobService/job-service)
- CDDRI Data Enrichment Worker
- CDDRI Production